content/syllabuses/employed-intro-to-tilde-post-skills-test.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Employed intro to Tilde for post skills test
+---
+
+- {{% contentlink path="projects/tilde/link-card-tutorial" flavour=""%}}
```

Comment:
They probably need the link tutrial first since they'll be linking us to things as part of their skills test.